base/nullable.jl

```diff
@@ -78,3 +78,30 @@ function hash(x::Nullable, h::UInt)
         return hash(x.value, h + nullablehash_seed)
     end
 end
+
+# indexing is either without index, or with 1 as index
+function getindex(x::Nullable)
+    @boundscheck x.isnull && throw(NullException())
```

Comment:
BTW, you can call `isnull` on any type now, so no need to `isa(Nullable, x)` (which you used above).